libs/blocks/icon-block/icon-block.css

```diff
@@ -6,11 +6,20 @@
 
 .dark .icon-block,
 .icon-block.dark {
-  color: #fff;
+  color: var(--color-white);
+  background-color: var(--color-black);
+  width: 90%;
```

Comment:
Why is this reduction needed for `.dark` variants?

Reply:
This width reduction is needed to accommodate the black background on the `.dark` variant that is "full-width" but not entirely full-width per the mockup.
<img width="831" alt="Screenshot 2024-08-06 at 12 08 10 AM" src="https://github.com/user-attachments/assets/0eeed4a1-98d4-4a35-ab04-9efd00ddc6cc">